client/pseudonymsys_ec.go

```diff
@@ -28,11 +29,13 @@ func (c *PseudonymsysClientEC) GenerateNym(userSecret *big.Int,
 	caCertificate *pseudonymsys.CACertificateEC) (
 	*pseudonymsys.PseudonymEC, error) {
 	// new client needs to be created in each method to implicitly call server Run method:
-	genericClient, err := newGenericClient(c.endpoint)
+	genericClient, err := newGenericClient(c.conn)
```

Comment:
Same here, please move struct initialization into the `NewPseudonymsysClientEC` constructor for all the functions:
```
GenerateNym
ObtainCredential
TransferCredential
```